Source/GenerateUserTypesFromPdb/GenerateUserTypesFromPdb.csproj

```diff
@@ -74,7 +75,7 @@
   </PropertyGroup>
   <PropertyGroup />
   <PropertyGroup>
-    <NoWin32Manifest>true</NoWin32Manifest>
+    <ApplicationManifest>app.manifest</ApplicationManifest>
```

Comment:
This cannot go into master branch if we aren't copying msdia140.dll to output folder...